setup.cfg

```diff
@@ -16,7 +16,7 @@ search = __version__ = "{current_version}"
 replace = __version__ = "{new_version}"
 
 [bdist_wheel]
-universal = 1
+universal = 0
```

Comment:
interesting, what would the platform definitions be to contain a potential binary wheel to a given release?